libraries/NativeEthernet/examples/ChatServer/ChatServer.ino

```diff
@@ -4,16 +4,32 @@
  A simple server that distributes any incoming messages to all
  connected clients.  To use, telnet to your device's IP address and type.
  You can see the client's input in the serial monitor as well.
- Using an Arduino Wiznet Ethernet shield.
 
  Circuit:
- * Ethernet shield attached to pins 10, 11, 12, 13
+ * STM32 board with Ethernet support
 
  created 18 Dec 2009
  by David A. Mellis
  modified 9 Apr 2012
  by Tom Igoe
 
+ Workaround:
+ * If you can't send a message after the server has accepted the connection,
+ reject the 26 first bytes.
+
```

Comment:
this message is not very clear. What is the exact issue and why do we reject 26 bytes  ?